### PR TITLE
docs: Update the intro tutorial with extended analysis

### PR DESCRIPTION
**Pull request by rht**

### CODE REVIEW

**Comment by rht** [comment]
I can't modify the branch, so I have to reopen this PR from my branch, with @EwoutH's commits intact. Reopened in #1717.

File: docs/index.rst [hunk 1]
```diff
@@ -96,7 +96,7 @@ ABM features users have shared that you may want to use in your model
 
    Mesa Overview <overview>
    tutorials/intro_tutorial
```
Comment (tpike3) [inline docs/index.rst:98]: Is there a reason `adv_tutorial` is `.ipynb` but `intro_tutorial` is not?

Comment (rht) [reply docs/index.rst:98]: I was adding minimal amount of changes to the baseline v1.2, because the PR modified only the intro tutorial.

Comment (tpike3) [reply docs/index.rst:98]: That's fair

**Pull request closed without merging**